packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -73,24 +98,23 @@ const DetailView = ({ title, subtitles, graph, details, footer, isDivided, ...cu
   }
 
   return (
-    <div {...attributes}>
-      <div className={cx('title')}>
-        <h1 className={cx('primary-text')}>{title}</h1>
+    <div {...attributes} className={attributes.className}>
+      <div className={cx('title', { 'title-smaller': isSmallerTitles })}>
+        {titleElement}
+        {secondaryTitlesElements}
         {subtitleElements}
+        {accessoryElement}
```

Comment:
I think the `accessoryElement` rendering should be moved outside of the `title` div

Reply:
@emilyrohrbough - That is a good note. We are using that div with the title class more as a wrapper around all elements in the top/title section, the accessory included, and has additional style with bottom-spacing on it, so changing the accessory position will affect the way the layout might need to be structured. 

It does look out of place, based on how the everything currently named, so I updated the class names from `title` to `titles-section` for now, and will try and rework the nesting order and the css with a future PR.